hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -280,7 +282,7 @@ class Lifter(handlerPaths: Opt[HandlerPaths])(using State, Raise):
       End()
     )
 
-    (defn, varsMap, varsList)
+    (defn, varsMap.mapValues(_.mapSecond(_.sym)).toMap, varsList)
```

Comment:
```
method mapValues in trait MapOps is deprecated since 2.13.0: Use .view.mapValues(f). A future version will include a strict version of this method (for now, .view.mapValues(f).toMap).bloop
```